spec/clang/spec_helper.cr

```diff
@@ -1,7 +1,7 @@
 require "../spec_helper"
 
 class ClangValidationError < Exception
-  getter document : JSON::Any::Type
+  getter document : JSON::Any::Type | UInt64
```

Comment:
Hey when does the | UInt64 case happen?

Reply:
This was my previous fix for the issue in #12 ive since reversed that

Reply:
removed